src/js/controller/Setup.js

```diff
@@ -62,11 +66,18 @@ define([
         ];
 
         this.start = function () {
+            _setupFailureTimeout = setTimeout(_setupTimeoutHandler, _errorTimeoutDelay * 1000);
```

Comment:
Put the * 1000 in the original place, ie

            _errorTimeoutDelay = 10 * 1000;